docs/settings.rst

```diff
@@ -1000,8 +1000,8 @@ the ``TAG_FEED_ATOM`` and ``TAG_FEED_RSS`` settings:
 
 .. data:: FEED_MAX_ITEMS
 
-   Maximum number of items allowed in a feed. Feed item quantity is
-   unrestricted by default.
+   Maximum number of items allowed in a feed. `False` for unrestricted. 100 if
```

Comment:
Another similar option `SUMMARY_MAX_LENGTH` uses `None` for disabling the cutoff. So it should be `None` here as well for consistency.